### PR TITLE
Added License to Readme

### PR DESCRIPTION
**Pull request by kartikeysingh6**

### CODE REVIEW

**Comment by seehait** [comment]
- Please rebase from the latest commit of master branch.
- Please add a link to LICENSE.md file instead.
- Please update table of content as well.

File: README.md [hunk 1]
```diff
@@ -34,3 +34,6 @@ series-renamers --directory [path/to/target/directory] --prefix 'Series S01 E' [
 | `--directory`, `-d` | `string`  | path to target directory | `false`  | current directory |
 | `--prefix`, `-p`    | `string`  | new file name prefix     | `true`   |                   |
 | `--dry-run`, `-dr`  | `boolean` | only preview the results | `false`  | `false`           |
+
+## License
+MIT License
```
Comment (seehait) [inline README.md:39]: Please add a link to LICENSE.md file.

Comment (kartikeysingh6) [reply README.md:39]: Will do

Comment (seehait) [reply README.md:39]:
```suggestion
*Series Renamer Tools* is licensed under  [MIT License](LICENSE)
```

**Comment by seehait** [comment]
The new one is created https://github.com/seehait/series-renamer-tools/pull/20

**Pull request closed without merging**